### PR TITLE
Add Travis CI integration

### PR DESCRIPTION
**Pull request by mlangbehn**
This pull request is comprised of commits 253c3ea, and 404b191. It adds Travis CI integration with shell script and YAML linting.

### CODE REVIEW

File: defaults.yml [hunk 1]
```diff
@@ -158,9 +158,9 @@ apt_repos:
   erlang:
     repo: http://packages.erlang-solutions.com/debian
     key_url: https://packages.erlang-solutions.com/debian/erlang_solutions.asc
-#  aptly:
-#    repo: https://apt-mirror.openstack.blueboxgrid.com/aptly/squeeze
-#    key_url: https://apt-mirror.openstack.blueboxgrid.com/keys/aptly.key
+  # aptly:
```
Comment (paulczar) [inline defaults.yml]: actually we could just delete these lines just in there from legacy copy/paste BS.

File: monitor/group_vars/all.yml [hunk 2]
```diff
@@ -72,18 +72,18 @@ sensu:
...
-#      flapjack:
-#        enabled: False
-#        host: 127.0.0.1
-#        port: 6380
-#      flapjack_http:
-#        enabled: True
-#        uri: http://127.0.0.1:3090
-#        ttl: -1 # -1 disables, otherwise httpbroker re-sends last received state until ttl expires
-#        service_owner: development
+  # handlers:
```
Comment (paulczar) [inline monitor/group_vars/all.yml]: we can nuke these as well, we don't need to flapjack for this demo so probably wouldn't uncomment them back in.

**Review by paulczar — CHANGES_REQUESTED** [review]
a couple of commented out portions we should just wipe instead of fixing the linting for.  otherwise :+1:

File: defaults.yml (hunk 1 above)
Comment (mlangbehn) [reply defaults.yml]: @paulczar: Updated.

File: monitor/group_vars/all.yml (hunk 2 above)
Comment (mlangbehn) [reply monitor/group_vars/all.yml]: @paulczar: Updated.

**Review by paulczar — APPROVED** [review]

**Pull request merged**